applications/ssotap/values-idfdev.yaml

```diff
@@ -1,4 +1,21 @@
 cadc-tap:
+  config:
+    uwsDatabaseUrl: "postgresql://localhost:5432/ssotap"
```

Comment:
If CloudSQL is in use, the only part of this that really is configurable is the database name. I wonder if it would make sense to say that the database name and user will always be the same for CloudSQL and move that, as a single setting, into the CloudSQL section, and then just set the database URL to this if CloudSQL is enabled. That would simplify the interface here a bit.

Reply:
I think that makes sense. One reason i had those as separate from the cloudsql configuration, was that I was considering the possibility that someone will want to use the chart with an external UWS database that is not on cloudsql. 

Having said that my proposed changes do not quite allow that, as these were used only when cloudsql is enabled. So I've gone with your suggestion for now, and if we want to add this capability we can reconsider which structure makes it easier and more transparent.